app/controllers/shift_commerce/static_pages_controller.rb

```diff
@@ -1,5 +1,5 @@
 module ShiftCommerce
-  class StaticPagesController < ShiftCommerce::ApplicationController
+  class StaticPagesController < ::ApplicationController
```

Comment:
Inherit from the possibly-customised parent application controller instead of the stock Shift one.